src/us/kbase/workspace/test/workspace/ObjectResolverTest.java

```diff
@@ -21,7 +21,6 @@
 import us.kbase.common.test.TestException;
```

Comment:
You'll notice there's quite a bit of repeated object creation code in this and other test classes. In test classes, I'm a lot less likely to want to DRY things up because you wind up having to jump all over the place trying to figure out where setup objects came from; to me it's easier to read if the test is as self-contained as possible and you don't have to read other methods to figure out what's going on. That said, if you find something particularly egregious I'm happy to discuss.

Reply:
If that's your preference as primary (only?) code editor and maintainer for this workspace, I am happy not to suggest any refactoring.